src/auro-sidenav.js

```diff
@@ -29,7 +29,12 @@ import styleCss from "./style-css.js";
 export class AuroSidenav extends LitElement {
 
   static get properties() {
-    return {};
+    return {
+      collapsible: {
```

Comment:
This should be specific to auro but general and reusable across components using this pattern.

Reply:
I propose we avoid 'mobile' or 'desktop' in our language to keep this generic. While not in our current use case, this _can_ be wrapped in an accordion at whichever width is required.

Reply:
@blackfalcon, @jason-capsule42, @jordanjones243 - I'd appreciate any feedback and/or suggestions regarding this variable name.

Reply:
My vote, `isToggle`.